src/openrct2/world/MapAnimation.cpp

```diff
@@ -443,6 +459,8 @@ static bool map_animation_invalidate_large_scenery(int32_t x, int32_t y, int32_t
 
     bool wasInvalidated = false;
     tileElement = map_get_first_element_at(x >> 5, y >> 5);
+    if (tileElement == nullptr)
+        return wasInvalidated;
```

Comment:
`return true;` Returning true deletes this animation so if it has bad data it wont be hit again.